.github/workflows/build.yml

```diff
@@ -116,3 +116,36 @@ jobs:
     - run: make clang-release-ubsan
     - run: make test-clang-debug-ubsan
     - run: make test-clang-release-ubsan
+
+  build-min-cmake:
+    name: min-cmake
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install Ninja
+      run: sudo apt-get install ninja-build
+    - name: Detect minimum CMake version
+      run: >
+        awk 'match($0, /cmake_minimum_required\(VERSION *([0-9]+\.[0-9]+)\)/, a)
+        { print "WABT_CMAKE_VER=" a[1] }' CMakeLists.txt | tee $GITHUB_ENV
+    - name: Install minimum CMake
+      run: |
+        python -m pip install -U setuptools wheel pip
+        python -m pip install "cmake==${WABT_CMAKE_VER}.*"
+        cmake --version
+        [[ "$(cmake --version)" =~ ${WABT_CMAKE_VER}. ]]
```

Comment:
I think this test may not quite capture it. E.g. if WABT_CMAKE_VER is "3.2", this will pass if `cmake --version` prints "cmake version 3.22.1". (I think the trailing "." is still part of the extended regex and matches any character.) On the other hand, if `cmake --version` actually does print "cmake version 3.2", this will fail (but should pass).

My shell scripting is not great, but what would you think about something like
```sh
cmake --version | head -n1 | grep -P "^cmake version \Q${REGEX}\E(\.\d+)?$"
```

Reply:
Yeah, that seems fine too. Shell scripting is so finicky.

Reply:
Fwiw my understanding was that setup-python put Python's bin directory early in the path. Reading the code suggests that is the case, though documentation is sparse.

Reply:
https://github.com/actions/toolkit/blob/bc4be505973a6a7344bfd71e1b32f77e1755310c/packages/core/src/core.ts#L124

https://github.com/actions/setup-python/blob/f6795b163d3422199fb74e169e9dc477edf891b1/src/find-python.ts#L125

Reply:
Ok, it does seem a bit fragile to rely on an undocumented PATH order, but the current behavior of that code looks good, and this isn't launching missiles so I think we can live with it. Up to you if you want to use the exact test or just remove the test. lgtm either way.

Reply:
I think I'll cut it. It might be undocumented, but path-prepending is so central to GHA's workflow ecosystem, I'm quite sure Hyrum's Law prevents them from ever changing it.

Reply:
Done!